code/api/src/modules/user/query.test.js

```diff
@@ -17,7 +18,43 @@ describe('user queries', () => {
         schema: schema,
         graphiql: false,
       }),
-    )
+    );
+    await models.User.destroy({ where: {} })
+  })
+
+  beforeEach(async () => {
+    const userData1 = {
+      id: 1,
+      name: "testUser1",
+      email: "test1@example.com",
+      password: bcrypt.hashSync('abc123', 10),
+      role: "USER",
+      createdAt: new Date(),
+      updatedAt: new Date(),
+      styleSummary: "coding cowboy"
+    };
+
+    const userData2 = {
+      id: 2,
+      name: "testUser2",
+      email: "test2@example.com",
+      password: bcrypt.hashSync('123abc', 10),
+      role: "USER",
+      createdAt: new Date(),
+      updatedAt: new Date(),
+      styleSummary: "goth farmer"
+    };
+
+    await models.User.create(userData1);
+    await models.User.create(userData2);
+  })
+
+  afterEach(async () => {
+    await models.User.destroy({ where: {} })
+  })
+
+  afterAll(() => {
```

Comment:
These functions will ensure that our tests are torn down properly in order to stop tests from "bleeding" into each other

Reply:
This is awesome!